meshtastic/tests/test_mesh_interface_behavioral_compat.py

```
@@ -0,0 +1,1829 @@
+            acknowledgment_attr, request_id=request_id, clear_scoped=True
+        )
+
+        # Add to active wait request IDs
+        with iface._response_handlers_lock:
+            iface._active_wait_request_ids.setdefault(acknowledgment_attr, set()).add(
+                request_id
+            )
+
+        # Add a response handler
+        iface._request_wait_runtime.add_response_handler(
+            request_id, lambda p: None, ack_permitted=True
```

Ruff (0.15.7)

[warning] Unused lambda argument: `p`

(ARG005)